### ISSUE TITLE
Aria tags for Framework/Appflow dropdown

### ISSUE THREAD

**Issue by perrygovier** [ionic-team/ionic-docs#315]
Most of the new site works pretty well with JS turned off so web crawlers should be able to navigate the site just fine. The only problem I can find is the framework/appflow dropdown doesn't w/o JS. That's understandable, but we should add Aria tags to help it out. Adding them to the main nav components is a good idea too.
